util/forwardedhosthandler_test.go

```diff
@@ -0,0 +1,61 @@
+package util
+
+import (
+	mockHttp "content-ui-service/mocks/net/http"
```

Comment:
i have added the generation script we use everywhere else and also a line in interfaces.go to generate the mock. the mock is there now too,.